src/afl-fuzz-run.c

```diff
@@ -851,17 +913,19 @@ void sync_fuzzers(afl_state_t *afl) {
 
     snprintf(id0, sizeof(id0), "%s/id:000000,*", qd_path);
 
-    if (glob(id0, 0, NULL, &glob_result) == 0 && glob_result.gl_pathc == 1 &&
-        stat(glob_result.gl_pathv[0], &st) == 0) {
+    if (likely(glob(id0, 0, NULL, &glob_result) == 0 &&
+               glob_result.gl_pathc == 1 &&
+               stat(glob_result.gl_pathv[0], &st) == 0)) {
 
       // we found exactly one "id:000000,*" file and obtained its mtime
       globfree(&glob_result);
 
-      if (last_mtime && last_mtime < st.st_mtime) {
+      if (unlikely(last_mtime && last_mtime < st.st_mtime)) {
```

Comment:
BTW, id:000000 is usually hard-linked of input file and thus its mtime is not the fuzzer start time.

Reply:
yeah true. Then I will use the `cmdline` file instead

Reply:
otherwise you think it is fine?

Reply:
Yes, looks good to me